certificate_agent.py

```diff
@@ -57,8 +57,8 @@ def main():
                                 settings.QUEUE_AUTH_USER,
                                 settings.QUEUE_AUTH_PASS,
                                 settings.QUEUE_USER, settings.QUEUE_PASS)
-    last_course = None  # The last course_id we generated for
-    cert = None  # A CertificateGen instance for a particular course
+    # Optimiztion to reuse previous template
```

Comment:
Updated the comment to be more helpful.  

Your first paragraph doesn't make sense.  I'm not sure what your getting at.  

The second paragraph is an argument we could use for any comment.  Remove all comments now so we don't forgot to remove them later?  Also, it is referencing the line directly below it not one 50 lines away.  It explains why the variable exists when it is not used outside of the while loop.